Error compiling getting-started-grpc

## In which file did you encounter the issue?
endpoints\getting-started-grpc\gradlew.bat

### Did you change the file? If so, how?
No

## Describe the issue

I am unable to build the _gRPC Getting Started_ sample on Windows with JDK 8.
I am running the following command:

```
gradlew.bat build
```

Here is the result:

```
FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\davidair\github\GoogleCloudPlatform\java-docs-samples\endpoints\getting-started-grpc\api\build.gradle' line: 18

* What went wrong:
A problem occurred evaluating project ':api'.
> Failed to apply plugin [id 'com.google.protobuf']
   > Could not create an instance of type com.google.protobuf.gradle.ProtobufSourceDirectorySet.
      > org.gradle.api.internal.file.DefaultSourceDirectorySet.<init>(Ljava/lang/String;Ljava/lang/String;Lorg/gradle/api/internal/file/FileResolver;Lorg/gradle/api/internal/file/collections/DirectoryFileTreeFactory;)V

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
```

OS: Windows 10 version 1903
JAVA_HOME=C:\Program Files\Java\jdk1.8.0_241

Note that I was able to successfully build a non-Cloud gRPC sample from at the latest release to get the example code:
$ git clone -b v1.28.0 https://github.com/grpc/grpc-java

--- a/endpoints/getting-started-grpc/api/build.gradle
+++ b/endpoints/getting-started-grpc/api/build.gradle
@@ -22,7 +22,7 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.0'
+    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.8'
 
   }
 }

